Bio/motifs/clusterbuster.py

```diff
@@ -58,21 +70,30 @@ def read(handle):
 
     motif = motifs.Motif(alphabet="GATC", counts=nucleotide_counts)
     motif.name = motif_name
+    motif.gap = motif_gap
+    motif.weight = motif_weight
     record.append(motif)
 
     return record
 
 
-def write(motifs):
+def write(motifs, precision=0):
     """Return the representation of motifs in Cluster Buster position frequency matrix format."""
```

Comment:
Can you briefly describe the precision argument here, with default 0 meaning integers.

Note there should be a blank line following the current one-line summary, and then the new text.

Reply:
done.

Reply:
Did you forget to push a commit?

Reply:
Yes I forgot. Force pushed now.

Reply:
The opening triple-quote and the one-line summary should be on the same line
https://peps.python.org/pep-0257/#multi-line-docstrings

That this appears to have passed the automated tests puzzles me... possibly a flake8 to ruff regression? i.e. Perhaps ruff misses this.

Reply:
> The opening triple-quote and the one-line summary should be on the same line https://peps.python.org/pep-0257/#multi-line-docstrings
> 
> That this appears to have passed the automated tests puzzles me... possibly a flake8 to ruff regression? i.e. Perhaps ruff misses this.

From that link:
> The summary line may be on the same line as the opening quotes **or on the next line**. The entire docstring is indented the same as the quotes at its first line (see example below).

Reply:
My memory may be at fault, but likely ruff takes a less strict interpretation that the flake8 based system we used to use.